packages/cli/src/commands/setup/tailwind/tailwind.js

```diff
@@ -37,23 +37,58 @@ const tailwindImportsAndNotes = [
 
 const INDEX_CSS_PATH = path.join(getPaths().web.src, 'index.css')
 
+const tailwindImportsDoesExist = (indexCSS) => {
```

Comment:
Would just simplify this function name (which fixes a grammatical error at the same time ^^)

```suggestion
const tailwindImportsExist = (indexCSS) => {
```